superset/assets/src/dashboard/reducers/getInitialState.js

```diff
@@ -6,12 +6,17 @@ import { initSliceEntities } from './sliceEntities';
 import { getParam } from '../../modules/utils';
 import { applyDefaultFormData } from '../../explore/store';
 import { getColorFromScheme } from '../../modules/colors';
-import layoutConverter from '../util/dashboardLayoutConverter';
+import findFirstParentContainerId from '../util/findFirstParentContainer';
+import layoutConverter, {
+  getEmptyLayout,
+  getRowContainer,
+  getChartHolder,
+} from '../util/dashboardLayoutConverter';
```

Comment:
could we break these into individual files?

Reply:
fixed.